### PR TITLE
Add Trusty, Xenial, Bionic, and Focal distributions to Travis CI (#39)

### PR DESCRIPTION
**Pull request by knbarton**
The goal of this is to solve issue #39.

### CODE REVIEW

**Comment by knbarton** [comment]
@bkochuna Do I need to add reviewers for this since I'm the only assignee? I _think_ I got it working. There are 4 jobs running, and for each job the 'build system information' shows a different distribution.

**Comment by bkochuna** [comment]
I’ll review

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,15 @@
 language: cpp
-os: linux
+
+jobs:
+  include:
+    - os: linux
```
Comment (bkochuna) [inline .travis.yml:5]: Why do you put os here with dist? Can you just put os outside the jobs and keep dist in the include?

Comment (knbarton) [reply .travis.yml:5]: I had been testing it with os both inside and outside jobs to see if it changed anything. I thought I had reverted it back to having os outside jobs, but I guess I didn't push it up. I'll fix that.

**Review by bkochuna — APPROVED** [review]
Looks Good!

**Pull request merged**